Dockerfile.sdk

```diff
@@ -173,6 +177,25 @@ RUN apt-get update && \
     pip3 install --upgrade grpcio-tools && \
     pip3 install --upgrade pip
 
+# Client build requires recent version of CMake (FetchContent required)
```

Comment:
I suggest moving the installation of these build deps to the test instead of here. This would install these in the final sdk container which should not contain build deps, only runtime deps

Reply:
Thanks for the suggestion. The installation of build deps are moved to the test. Please review.